Jenkinsfile

```diff
@@ -81,6 +83,9 @@ pipeline {
                         }
                     }
                 }
+                script {
+                    sh 'curl -s https://codecov.collibra.com/bash | bash -s -- -c -f _output/tests/**/coverage.txt -F unittests'
```

Comment:
It this URL any different from https://codecov.io/bash? I haven't seen it before.

Reply:
I got this from the codecov docs site: https://docs.codecov.io/docs/supported-ci-providers.